Makefile

```diff
@@ -98,7 +100,7 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: manager
-	docker build -t ${IMG} .
+	docker build --build-arg COMMIT=${GIT_COMMIT} --build-arg DATE=${BUILD_DATE} -t ${IMG} .
```

Comment:
It would be better build on release, or tag. We dont want to flush docker repo.

Reply:
Build args should only affect the golang run command which if your running `make docker-build` this is expected.